cmd/process-agent/app/subcommands/taggerlist/tagger_list.go

```diff
@@ -7,33 +7,49 @@ package app
 
 import (
 	"fmt"
-	"os"
+	"go.uber.org/fx"
```

Comment:
format: this probably belongs to the second import group

Reply:
Updated.